README.md

```diff
@@ -1,29 +1,63 @@
 # Evergreen.py
 
 A client library for the Evergreen API written in python. Currently supports the V2 version of
-the API. For more details, see https://github.com/evergreen-ci/evergreen/wiki/REST-V2-Usage .
+the API. For more details, see https://github.com/evergreen-ci/evergreen/wiki/REST-V2-Usage.
 
 ![PyPI - Python Version](https://img.shields.io/pypi/pyversions/evergreen.py) [![PyPI](https://img.shields.io/pypi/v/evergreen.py.svg)](https://pypi.org/project/evergreen.py/) [![Coverage Status](https://coveralls.io/repos/github/evergreen-ci/evergreen.py/badge.svg?branch=master)](https://coveralls.io/github/evergreen-ci/evergreen.py?branch=master)
 
-## Documentation
+## Table of contents
 
-You can find the documentation [here](https://evergreen-ci.github.io/evergreen.py/).
+1. [Description](#description)
+2. [Dependencies](#dependencies)
+3. [Installation](#installation)
+4. [Usage](#usage)
+5. [Documentation](#documentation)
+6. [Contributor's Guide](#contributors-guide)
+    - [High Level Architecture](#high-level-architecture)
```

Comment:
It looks like some of these sub-tasks missing. We should either add them, or remove them from this list.